pkg/vpcmodel/nodesExplainability.go

```diff
@@ -639,15 +683,33 @@ func (v *VPCConnectivity) getConnection(c *VPCConfig, src, dst Node) (conn *comm
 	return conn, nil
 }
 
-func (rulesInLayers rulesInLayers) string(c *VPCConfig) string {
+// prints either rulesDetails by calling StringDetailsRulesOfFilter or effect of each filter by calling StringFilterEffect
+func (rulesInLayers rulesInLayers) string(c *VPCConfig, isIngress, rulesDetails bool) string {
 	rulesInLayersStr := ""
-	for _, layer := range filterLayers {
+	// order of presentation should be same as order of evaluation:
+	// (1) the SGs attached to the src NIF (2) the outbound rules in the ACL attached to the src NIF's subnet
+	// (3) the inbound rules in the ACL attached to the dst NIF's subnet (4) the SGs attached to the dst NIF.
+	// thus, egress: security group first, ingress: nacl first
+	filterLayersOrder := filterLayers
```

Comment:
do not relay on the order of filterLayers
```suggestion
	filterLayersOrder := [2]string{SecurityGroupLayer, NaclLayer}
	if isIngress {
        	filterLayersOrder = [2]string{NaclLayer, SecurityGroupLayer}
	}
```

Reply:
I need to print in a specific order - how do you suggest to do it?

Reply:
see my code suggestion above. (filterLayersOrder := [2]string{SecurityGroupLayer, NaclLayer})

Reply:
Thanks :-)